### DISCUSSION TITLE
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested

### DISCUSSION THREAD

**Discussion by panikajo** [aahnik/tgcf#488] [General]
root@gamepanel:~# docker pull aahnik/tgcf
Using default tag: latest
latest: Pulling from aahnik/tgcf
f2f58072e9ed: Pull complete
5c8cfbf51e6e: Pull complete
aa3a609d1579: Pull complete
094e7d9bb04e: Pull complete
2cbfd734f382: Pull complete
aa86ac293d0f: Pull complete
9a8a815ca9ef: Pull complete
6b61f725b1ab: Pull complete
b846c6cbfd9a: Pull complete
89d76304d6d4: Pull complete
f9306bf87235: Pull complete
9adca84113e8: Pull complete
5491cd0ab6c7: Pull complete
59eb089b885e: Pull complete
e530212988d2: Pull complete
Digest: sha256:6fad299b6a46acd5902abd5e45c8c662a739003d83b393bb4a911f7a2f4a1c3a
Status: Downloaded newer image for aahnik/tgcf:latest
docker.io/aahnik/tgcf:latest

root@gamepanel:~# docker run -d -p 8501:8501 --env-file .env aahnik/tgcf
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested

**Comment by MobCode100** [comment]
If it works fine, then you can safely ignore the warning.

**Reply by panikajo** [reply]
it doesn't even start.
If you register docker ps then there is nothing

![image](https://github.com/aahnik/tgcf/assets/4964185/91700074-fbcc-4da4-a2c2-0b30a40742f5)

**Comment by panikajo** [comment]
> If it works fine, then you can safely ignore the warning.
